crates/ruff_linter/src/rules/flake8_pytest_style/types.rs

```diff
@@ -24,7 +24,7 @@ impl Default for ParametrizeNameType {
 impl Display for ParametrizeNameType {
     fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
         match self {
-            Self::Csv => write!(f, "csv"),
+            Self::Csv => write!(f, "comma-separated list string"),
```

Comment:
```suggestion
            Self::Csv => write!(f, "comma-separated list of strings"),
```

Reply:
Done

Reply:
Although I'm still not happy with this. To me a "comma-seperated list of strings" would be `['string1', 'string2']`

Reply:
Maybe "string of comma-seperated values"

Reply:
Oh I see. Yes, "string of comma-separated values" works well, I think! Good call.

Reply:
Ok done

Reply:
Thanks!